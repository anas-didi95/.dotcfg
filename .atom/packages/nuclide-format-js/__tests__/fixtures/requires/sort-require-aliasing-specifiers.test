

const {b: C, a: d} = require('Y');
const {c} = require('z');

c(d(C, X));
