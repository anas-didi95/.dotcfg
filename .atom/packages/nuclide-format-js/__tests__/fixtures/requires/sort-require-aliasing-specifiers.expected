const X = require('X');

const {a: d, b: C} = require('Y');
const {c} = require('z');

c(d(C, X));
